anomalib/models/reverse_distillation/torch_model.py

```diff
@@ -68,6 +68,8 @@ def forward(self, images: Tensor) -> Union[Tensor, Tuple[List[Tensor], List[Tens
             Union[Tensor, Tuple[List[Tensor],List[Tensor]]]: Encoder and decoder features in training mode,
                 else anomaly maps.
         """
+        self.encoder.eval()
```

Comment:
Are the benchmarking results still the same after this change?

Reply:
I am running it right now. But that was a good catch. I saw that the encoder was in training mode in the train step.

Reply:
Interestingly apart from pixel level score, image level scores are lower 
![image](https://user-images.githubusercontent.com/17232914/172832983-374f1f15-66d1-4081-b93b-71eec6f27bc8.png)
![image](https://user-images.githubusercontent.com/17232914/172833009-d0d7eb98-ed75-4685-82cf-0846254522c6.png)

Reply:
The average results also seem to be a bit lower than those reported in the original paper (98.5% image AUROC, 97.8% pixel AUROC). For now I would suggest to merge this PR, but it would be good to investigate if this difference can be explained.